src/components/app/Home.js

```diff
@@ -117,9 +116,10 @@ class ActionButtons extends React.PureComponent<
             type="button"
             className={classNames({
               homeSectionButton: true,
-              homeSectionButtonPressed: this.state.isLoadFromUrlPressed,
             })}
             onClick={this._loadFromUrlPressed}
+            // to make this a toggle button
+            aria-pressed={this.state.isLoadFromUrlPressed}
```

Comment:
I wonder if we shouldn't use `aria-expanded` instead (or in addition ?)

Reply:
I've done more research and decided to go with aria-expanded, also adding aria-controls (as per [here](https://www.deque.com/blog/accessible-aria-buttons/). Let me know what you think :)